etl/steps/data/grapher/homicide/2024-10-30/unodc.py

```diff
@@ -0,0 +1,24 @@
+"""Load a garden dataset and create a grapher dataset."""
+from etl.helpers import PathFinder, create_dataset
+
+# Get paths and naming conventions for current step.
+paths = PathFinder(__file__)
+
+
+def run(dest_dir: str) -> None:
+    #
+    # Load garden dataset.
+    ds_garden = paths.load_dataset("unodc")
+
+    table_names = ds_garden.table_names
+    # if your data is in long format, you can use `grapher_helpers.long_to_wide_tables`
+    # to get into wide format
+    tables = []
```

Comment:
I'm surprised we don't have a dataset method that returns all tables at once - should this exist maybe?

Reply:
Good point, I don't _think_ we do, but I'm not very on top of all the helper functions. Could be a good thing to add in a cooldown :)